src/qilib/utils/storage/mongo.py

```diff
@@ -208,6 +262,43 @@ def save_data(self, data: Any, tag: TagType) -> None:
         StorageInterface._validate_tag(tag)
         self._store_value_by_tag(tag, self._encode_data(self._serialize(data)), self._get_root())
 
+    def load_individual_data(self, field: Any, tag: TagType) -> Any:
+        """ Retrieve an individual field value at a given tag
+
+            Args:
+                field: Name of the individual field to be retrieved
+                tag: The tag
+
+            Returns:
+                Value of the field
+        """
+
+        StorageInterface._validate_tag(tag)
+
+        if len(tag) == 0:
+            raise NoDataAtKeyError('Tag cannot be empty')
+
+        encoded_field_name = self._encode_data(self._serialize(field))
+
+        return self._unserialize(self._decode_data(
+            self._retrieve_individual_value_by_tag(encoded_field_name, tag, self._get_root())))
+
+    def update_individual_data(self, field: Any, data: Any, tag: TagType) -> None:
```

Comment:
I'm starting to doubt if field should be Any. What does it mean when field is a  list of floats or a dictionary for example?